Vizzuality/half-earth-v3
# Add searchType and use countries search for NRC

## Use countries search for NRC
### Description
We were using the lookup table search for NRC and we decided that is best to go back to only country search
### Testing instructions
Go to NRC page and search. You should get only countries. The full and simple searches should also work ok
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-557?atlOrigin=eyJpIjoiZWZhM2ZlNzNiMzk3NGI4NGJmMTk0MGJiNTM2NTMwZTMiLCJwIjoiaiJ9
